assets/dev/js/frontend/handlers/base-nested-tabs.js

```diff
@@ -253,12 +253,12 @@ export default class BaseNestedTabs extends Base {
 	changeActiveTab( tabIndex, fromUser = false ) {
```

Comment:
I would prefer to use the `changeActiveTab()` function from the mega menu, which is much cleaner.